### DISCUSSION TITLE
How to use exceptions/error messages in filament?

### DISCUSSION THREAD

**Discussion by DevMushref** [filamentphp/filament#5800] [Help]
I have an app in production. Every action (with errors) I make, it just shows me a popup with `500` error after submitting, editing or deleting data.

Where can I find a documentation for filament to show errors in the form instead of 500 error?

**Comment by danharrin** [comment]
It's not Filament. Ensure that your Laravel app's .env has debug mode enabled.

**Reply by DevMushref** [reply]
Thank you for your response. But isn't that risky? If I enable the debug mode it will show the error and unnecessary information to the user.

**Reply by danharrin** [reply]
Validation errors in the form are always displayed. They aren't 500 errors.

500 errors come from parts of your code that are broken. You do not enable debug mode in production. Locally, debug mode should be abled so you can see what the errors are and fix them.
